var/spack/repos/builtin/packages/package.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Vpfft(MakefilePackage):
+    """ Proxy Application. VPFFT is an implementation of a mesoscale
+    micromechanical materials model. By solving the viscoplasticity
+    model, VPFFT simulates the evolution of a material under deformation.
+    The solution time to the viscoplasticity model, described by a set
+    of partial differential equations, is significantly reduced by the
+    application of Fast Fourier Transform in the VPFFT algorithm.
+    """
+
+    homepage = "http://www.exmatex.org/vpfft.html"
+    url      = "https://github.com/exmatex/VPFFT/archive/master.tar.gz"
+    tag      = ['proxy-app']
+
+    version('master', git='https://github.com/exmatex/VPFFT.git')
+
+    depends_on('eigen')
+    depends_on('fftw')
+    depends_on('mpi')
+
+    @property
+    def build_targets(self):
+        targets = [
+            "--file=Makefile.make",
+            "EIGEN_PATH={0}/include/eigen3".format(self.spec['eigen'].prefix),
```

Comment:
It will only be `eigen3` for certain versions of eigen. I would use:
```python
'EIGEN_PATH={0}'.format(join_path(self.spec['eigen'].prefix.include, 
    'eigen{0}'.format(self.spec['eigen'].version.up_to(1)))),
```